src/bundle/Resources/public/scss/_warning-icon.scss

```diff
@@ -0,0 +1,23 @@
+.ez-tabs {
+    .ez-warning-icon {
+        position: absolute;
+        z-index: 2;
```

Comment:
Why do you need z-index? In other places, with warning icon, we don't need it. We should avoid z-index